lnrpc/rpc.proto

```diff
@@ -2232,6 +2232,9 @@ message InvoiceHTLC {
 
     /// Current state the htlc is in.
     InvoiceHTLCState state = 8 [json_name = "state"];
+
+    /// The total amount of the mpp payment in msat.
+    uint64 total_amt_msat = 9 [json_name = "total_amt_msat"];
```

Comment:
also add `payment_addr`? could also add the `mpp_` prefix to match the sender side

Reply:
Good question. Payment address is already a property on the invoice level. Maybe we want to expose everything that we got from the htlc here. But on the other hand, htlcs with a payment address that doesn't match aren't added here. We are only recording failed htlcs after they got accepted initially.

I added the `mpp_` prefix

Reply:
good point, yes the payment addr should be referenced from the invoice itself 👍